packages/plugins/src/plugins/aave/helpers.ts

```diff
@@ -102,10 +111,13 @@ export async function fetchLendingForAddress(
       data: elementData,
       name: config.elementName,
       proxyInfo,
-    };
-    elements.push(element);
-  }
-  return elements;
+    } as PortfolioElementBorrowLend;
+  });
+
+  const elements = await Promise.all(elementPromises);
+  return elements.filter(
+    (element): element is PortfolioElementBorrowLend => element !== null
```

Comment:
We can get rid of `null` here too and simplify the above code.

```suggestion
    (element): element is PortfolioElementBorrowLend => !!element
```

Reply:
Unless I missed something ?

(and this is why you used `is PortfolioElementBorrowLend` to apply an _additional filter_ ?)

Reply:
element is PortfolioElementBorrowLend is because TS does not infer the type from filter. its finally fixed in TS 5.5

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-5.html